index.js

```diff
@@ -146,45 +173,52 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
+let ticketNumber = 1;
+function takeATicketNumber(line) {
+  line.push(ticketNumber);
+  console.log(`Welcome. You are ticket number ${ticketNumber}`);
+  ticketNumber += 1
+}
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+console.log(takeATicketNumber(line))
+// => `Welcome. You are ticket number 1`
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 2`
+console.log(takeATicketNumber(line))
+// => `Welcome. You are ticket number 2`
 
-// console.log(nowServing(line))
-// // => `Currently serving 1.`
+console.log(nowServing(line))
+// => `Currently serving 1.`
 
-// console.log(nowServing(line))
-// // => `Currently serving 2.`
-// console.log("%c----------", "color: red") 
+console.log(nowServing(line))
+// => `Currently serving 2.`
+console.log("%c----------", "color: red") 
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 3`
+console.log(takeATicketNumber(line))
+// => `Welcome. You are ticket number 3`
 
 
 // ***** Scope & Closures - Question 2 *****
-
+function ticketNumberGeneratorFunc() {
+  return takeATicketNumber
+}
```

Comment:
You're on the right track here with the function returning a function, but one of the other keys to this was removing the need to have a global variable. Here's the solution using a closure: 

```js
function ticketNumberGeneratorFunc() {
  let ticketNumber = 0

  function takeANumber(line) {
    ticketNumber++
    line.push(ticketNumber)
    return `Welcome. You are ticket number ${ticketNumber}`
  }

  return takeANumber
}
```